### ISSUE TITLE
Drop-down menus detect click release as a click

### ISSUE THREAD

**Issue by alexander213213** [godotengine/godot#80572]
### Godot version

4.1.1

### System information

OS: Ubuntu 22.04 jammy, Kernel: x86_64 Linux 6.2.0-26-generic

### Issue description

Some drop-down menu treats click release as a click which makes it hard to use a touchpad. The easiest to find is when I click the language menu in the Project Manager.

### Steps to reproduce

Click the language drop-down menu in the Project Manager window

### Minimal reproduction project

N/A

**Comment by lostptr** [comment]
Hey I managed to reproduce that with a few menus. I think all of them are like that, I couldnt find one that did not have this behavior. Just to be clear, you were hoping that the click event would be `button_down` rather than `button_up`?

**Comment by alexander213213** [comment]
Yes, the button_up makes it very difficult to work with a touchpad since upon clicking, people always do a button_up to move the mouse again

**Comment by Calinou** [comment]
Are you on a X11 or Wayland session? Use `echo $XDG_SESSION_TYPE` in a terminal to know. If on Wayland, try switching to a X11 session and see if you can reproduce this.

Also, try to reproduce this after enabling **Single Window Mode** in the Editor Settings then restarting the editor.
